Gemfile.lock

```diff
@@ -61,11 +61,24 @@ GEM
       msgpack (~> 1.0)
     builder (3.2.3)
     byebug (11.0.1)
+    cancan (1.6.10)
     coderay (1.1.2)
     concurrent-ruby (1.1.5)
+<<<<<<< HEAD
```

Comment:
was it fixed after the merge?

Reply:
Not yet, but I will fix. Sorry, I didn't notice that.